pkgs/by-name/be/beammp-launcher/package.nix

```
@@ -0,0 +1,69 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  cmake,
+  pkg-config,
+  boost,
+  httplib,
+  openssl,
+  nlohmann_json,
+  curl,
+  makeDesktopItem,
+  copyDesktopItems,
+  installShellFiles,
+}:
+stdenv.mkDerivation (finalAttrs: {
+  pname = "beammp-launcher";
+  version = "2.4.0";
```

The launcher has since been updated.

```suggestion
  version = "2.5.1";
```